.changeset/silver-points-cheer.md

```diff
@@ -0,0 +1,25 @@
+---
+'@clerk/chrome-extension': minor
+---
```

Comment:
❓ Is there any specific reason having two changesets?

Reply:
@octoper Yeah, it was back when changeset releases weren't working. I can remove the one.